jsonObject.class.asp

```diff
@@ -1114,6 +1117,62 @@ class JSONarray
 end class
 
 
+class JSONscript
+	dim i_parent, i_depth, i_version

```

Comment:
Depth and parent reference are not needed and can be removed.

Reply:
I wasn't quite sure if they were used from the parent object, so that's why I left it in. But it's removed now.